ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -72,7 +71,7 @@ Control {
 
         SortFilterProxyModel {
             id: communitiesStoreModel
-            sourceModel: root.profileStore.profileShowcaseCommunitiesModel
+            sourceModel: root.profileStore.showcasePreferencesCommunitiesModel
```

Comment:
@alexjba @micieslak @noeliaSD these models should be left-joined, same as for preferences